.pre-commit-config.yaml

```diff
@@ -0,0 +1,31 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.4.0
+    hooks:
+      - id: check-ast
+      - id: check-merge-conflict
+      - id: check-yaml
+      - id: debug-statements
+      - id: end-of-file-fixer
+        types: [python]
+      - id: flake8
+        args: ["--isolated", "--ignore=A001, P101, P103, W503"]
+        additional_dependencies:
+          - flake8-string-format
+          - flake8-comprehensions
+          - flake8-builtins
+      - id: requirements-txt-fixer
+      - id: trailing-whitespace
+        types: [python]
+
+  - repo: https://github.com/psf/black
+    rev: 19.10b0
```

Comment:
This will have to be updated regularly. Is there a way to automate the updating?

Reply:
Yes, there's.  I will set this to be automatically updated to the last stable version but keep in mind that this can change from one commit to another, as new version are released.